examples/plugins/date/main.go

```diff
@@ -49,5 +54,23 @@ func DateCommand() (*cobra.Command, error) {
 		},
 	}
 
+	// var r1 validator.Rule = &validator.Length{
+	// 	Limits: map[string]validator.Limit{
+	// 		"Use":   {Min: 1, Max: 5},
+	// 		"Short": {Min: 4, Max: 10},
+	// 		"Long":  {Min: 5, Max: 20},
+	// 		"Not":   {Min: 2, Max: 22},
+	// 	},
+	// }
+
+	// errr := r1.Validate(cmd, true)
+	// fmt.Println(errr)
+
+	var r2 validator.Rule = &validator.MustPresent{
```

Comment:
Why we run validator as part of the command?

Reply:
I did this just to test it's working